api/inc/register_gateway.h

```diff
@@ -19,13 +19,17 @@
 
 #include "api/inc/register_gateway_exports.h"
 #include "api/inc/uvisor_exports.h"
+#include "api/inc/svc_exports.h"
 #include <stdint.h>
 
 /** Get the offset of a struct member.
  * @internal
  */
 #define __UVISOR_OFFSETOF(type, member) ((uint32_t) (&(((type *)(0))->member)))
 
+/** Generate the SVCall opcode from the SVC ID. */
+#define UVISOR_SVC_OPCODE(id) ((uint16_t) 0xDF00 | (uint8_t) ((id) & 0xFF))
```

Comment:
@Patater the (uint8_t) caste in combination and the 0xFF seems to be too much - either one or the other?